app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,7 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.USE_BIOMETRIC" />
 
+    <!-- todo: remove usesCleartextTraffic after switching to https on NotificationService -->
```

Comment:
add number of task please 
and create it not to forgot about this!

Reply:
We already have it https://p2pvalidator.atlassian.net/browse/PWN-3208